docs/guides-agent-app.md

```diff
@@ -0,0 +1,485 @@
+
+This confirms that the Agent app has been assigned permissions and is now an app in the DAO.
+
+Note that **0x9ac98dc5f995bf0211ed589ef022719d1487e5cb2bab505676f0d084c07cf89a** is just the general identifier for the version of Agent app you have installed.
+
+You can also verify that permissions have been set properly through the UI:
+
+1. Click on the **Permissions** menu option in the left panel. You should see the Agent app at the end of the second row. Click on it.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-5.png">
+</p>
+
+2. Under **Actions available on this app** should see that the Voting app now manages who has the ability to execute actions for the Agent app. Scroll down.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-6.png">
+</p>
+
+3.  Under **Permissions set on this app** you should see that Voting app is also allowed to execute actions on behalf of the Agent app.
+
+<p align="center">
+   <img width="800" src="/docs/assets/agent-guide/agent-7.png">
+</p>
+
+# Use cases
+
+## A. Voting in another organization
+
+### **1. Create another Democracy DAO**
+
+Exactly the same as before, head over to [Aragon](https://rinkeby.aragon.org/) and choose the following defaults:
+
+- Support: 100%
+- Min. Quorum: 0%
+- Duration: 168 hours (or 1 week)
+
+### **2. Mint a token to allow our first DAO (A) to vote in our new DAO (B)**
+
+We've now created two Democracy DAOs -- let's call them **A** and **B**. A has an Agent app, B doesn't. We want to allow A to vote in B.
+
+Remember that A needs to be a tokenholder of B to be able to vote in B. And that A's Agent app acts its external interface.
+
+In other words, A's Agent app allows it to participate as a stakeholder in B.
+
+This means that to allow A to vote in B we need to mint a token for A's Agent app in B. 
+
+To do this run:
+
+```
+dao exec <name of dao B> <token manager app address of dao B> mint <agent app address of dao A> 1000000000000000000 --environment aragon:rinkeby --apm.ipfs.rpc https://ipfs.eth.aragon.network/ipfs/
+```
+
+Remember, you can find the addresses of the apps in any of your DAOs by running `dao apps <organization name> --environment aragon:rinkeby`.
+
+As you can see, we are using the `dao exec` command to interact with the `mint` method of B's [Token Manager](https://wiki.aragon.org/dev/apps/token-manager/) app.
+
+ [`mint`](https://wiki.aragon.org/dev/apps/token-manager/index.html#mint-tokens) is used to create new tokens and assign them to a receiver. It takes two arguments: a receiver address and the amount of tokens to be created.
+
+In our case the receiver is A's Agent App, and the amount of tokens to be created is 1.
+
+However, you should notice that instead of writing `1` as the second argument to `mint` we've gone with `1000000000000000000`.
+
+This is because a token has 18 decimals, so 1 unit of a token is actually 0.000000000000000001 tokens. This is not what we want.
```

Comment:
Tokens can have any number of decimals, the token created by the democracy template have 18 decimals